README.md

```diff
@@ -388,33 +418,40 @@ DEPLOYER_PK=
 MNEMONIC=
 ```
 
-Only either the `DEPLOYER_PK` or `MNEMONIC` is needed.
+Only one of the following is needed: 
+* `DEPLOYER_PK` 
+* `MNEMONIC`
 
 # Development
 
-This repository's development framework uses [HardHat](https://hardhat.org/getting-started/).
+This repository's development framework uses [Hardhat](https://hardhat.org/getting-started/).
 
-Tests are run using `npm run test`, more development tasks can be found in the package.json scripts.
+You can run the tests using `npm run test` and find more development tasks in the `package.json` scripts.
 
 ### Scripts
 
+You can install dependencies, test your project, check source code, and build your project with the commands below:
+
 ```sh
 npm install
 npm test
 npm run lint
 npm run build
+```
 
-# See Deployment section for more info
+For more information on the commands below, see the [Deployment](#deployment) section:
+```sh
 npx hardhat deploy:token
 npx hardhat deploy:factory
 npx hardhat deploy:token:impl
 ```
 
 ## Subgraph
 
-Check out our [Token Registry Subgraph](https://github.com/Open-Attestation/token-registry-subgraph) Github repository
+Check out the [Token Registry Subgraph GitHub repository](https://github.com/Open-Attestation/token-registry-subgraph)
 for more information on using and deploying your own subgraphs for the Token Registry contracts.
 
-## Notes
+## Additional information
 
-- The contracts have not gone through formal audits yet. Please use them at your own discretion.
+The contracts have not gone through formal audits. Use them at your own discretion.
+<!--Flag: Question here: Have the contracts gone thru formal audits so far? If not, shall we move this paragraph to the beginning to inform the user before they get started?-->
```

Comment:
To my knowledge no, but I think we can leave it here at the bottom as we don't want to scare potential users away